201-logic-app-veter-pipeline/azuredeploy.json

```diff
@@ -0,0 +1,220 @@
+      "metadata": {
+        "description": "The name of the function."
+      }
+    },
+    "integrationAccountName": {
+      "type": "string",
+      "defaultValue": "IntegrationAccount",
+      "minLength": 1,
+      "maxLength": 80,
+      "metadata": {
+        "description": "Name of the Integration Account."
+      }
+    },
+    "logicAppName": {
+      "type": "string",
+      "defaultValue": "VETERPipeline",
+      "minLength": 1,
+      "maxLength": 80,
+      "metadata": {
+        "description": "Name of the Logic App."
+      }
+    },
+    "logicAppLocation": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "allowedValues": [
+        "[resourceGroup().location]",
+        "australiaeast",
+        "australiasoutheast",
+        "brazilsouth",
+        "centralus",
+        "eastasia",
+        "eastus",
+        "eastus2",
+        "japaneast",
+        "japanwest",
+        "northcentralus",
+        "northeurope",
+        "southcentralus",
+        "southeastasia",
+        "westeurope",
+        "westus"
+      ],
+      "metadata": {
+        "description": "Location of the Logic App."
+      }
+    }
+  },
+  "variables": { },
+  "resources": [
+    {
+      "properties": { },
+      "sku": {
+        "name": "Free"
+      },
+      "name": "[parameters('integrationAccountName')]",
+      "type": "Microsoft.Logic/integrationAccounts",
+      "location": "[parameters('logicAppLocation')]",
+      "apiVersion": "2015-08-01-preview"
+    },
+    {
+      "properties": {
+        "schemaType": "xml",
+        "content": "<?xml version=\"1.0\" encoding=\"utf-8\"?><xs:schema xmlns=\"http://Integration.Order\" xmlns:b=\"http://schemas.microsoft.com/BizTalk/2003\" targetNamespace=\"http://Integration.Order\" xmlns:xs=\"http://www.w3.org/2001/XMLSchema\"><xs:element name=\"Order\"><xs:complexType><xs:sequence><xs:element name=\"Orderheader\"><xs:complexType><xs:sequence><xs:element name=\"OrderDate\" type=\"xs:string\" /><xs:element name=\"EstimatedDeliveryDate\" type=\"xs:string\" /><xs:element name=\"OrderNumber\" type=\"xs:string\" /></xs:sequence></xs:complexType></xs:element><xs:element name=\"CustomDetails\"><xs:complexType><xs:sequence><xs:element name=\"Name\" type=\"xs:string\" /><xs:element name=\"Address\" type=\"xs:string\" /></xs:sequence></xs:complexType></xs:element><xs:element name=\"OrderDetails\"><xs:complexType><xs:sequence><xs:element name=\"ItemDescription\" type=\"xs:string\" /><xs:element name=\"ItemCustomerCode\" type=\"xs:string\" /><xs:element name=\"TotalAmount\" type=\"xs:string\" /><xs:element name=\"UnitType\" type=\"xs:string\" /></xs:sequence></xs:complexType></xs:element></xs:sequence></xs:complexType></xs:element></xs:schema>",
+        "contentType": "application/xml"
+      },
+      "name": "[concat(parameters('integrationAccountName'), '/', 'Order')]",
+      "type": "Microsoft.Logic/integrationAccounts/schemas",
+      "apiVersion": "2015-08-01-preview",
```

Comment:
@gatneil Integration Account is still in preview. So we don't have a non-preview version right now.